binder/environment.yml

```diff
@@ -1,4 +1,4 @@
-name: example-environment
+name: mesa-tutorials
```

Comment:
please keep PRs focussed, this is a separate issue and thus does not belong in this PR. I suggest reverting all changes to this file